index.js

```diff
@@ -1,22 +1,18 @@
-'use strict'
+const express = require('express');
+const session = require('express-session');
+const crypto = require('crypto');
+const request = require('request');
+const jwt = require('jsonwebtoken');
+const jwkToPem = require('jwk-to-pem');
+require('dotenv').config();
```

Comment:
Could you please change this line to `require('dotenv').config({silent: true});` (I know it was already that way, but reviewing the code I just realized that if we don't have a `.dotenv` file it will throw an exception)